### PR TITLE
What is a failpoint?

### PR DESCRIPTION
**Pull request by lookfirst**

### CODE REVIEW

**Review by amyangfei — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [![Coverage Status](https://codecov.io/gh/pingcap/failpoint/branch/master/graph/badge.svg)](https://codecov.io/gh/pingcap/failpoint)
 [![Mentioned in Awesome Go](https://awesome.re/mentioned-badge.svg)](https://github.com/avelino/awesome-go)  
 
-An implementation of [failpoints][failpoint] for Golang.
+An implementation of [failpoints][failpoint] for Golang. Fail points are used to add code points where errors may be injected in a user controlled fashion. Fail point is a code snippet that is only executed when the corresponding failpoint is active.
```
Comment (kennytm) [inline README.md]: @lonng should the concept be spelled as "fail point" (2 words) or "failpoint" (1 word)?

Comment (lonng) [reply README.md]: Prefer to failpoint. How do you think about it?

Comment (kennytm) [reply README.md]: LGTM

**Pull request merged**